### PR TITLE
Stop superfluous copy of ld-linux so

### PR DESCRIPTION
**Pull request by Juuliuus**
deployELF function was copying ld-linux so from locations other than /lib64/. Use a similar routine in deployELF as is used in patchRpathsInElf to prohibit ld-linux so processing. #76

### CODE REVIEW

**Comment by probonopd** [comment]
Are you sure it's not needed in _both_ locations (e.g., because it was in _both_ locations on the original system)?

**Comment by Juuliuus** [comment]
No, I'm not sure. What I do know: 

This fix was used in all my testing and the resulting appimage worked flawlessly. 

At the time I noticed the problem I tracked down the "extra" ld-linux. 
It had copied /usr/lib64/ld-linux-x86-64.so.2  to appdir/usr/lib64 (this is the superfluous copy).

There were two other d-linux's as well.. /lib64/ld-linux-x86-64.so.2 (the one copied and patched by appimagetool), and /lib/x86_64-linux-gnu/ld-linux-x86-64.so.2.

All 3 of these ld-linux's are links to the same 187.0 KiB file: /lib/x86_64-linux-gnu/ld-2.31.so

In addition appimagetool writes this line (which comes out of patchRpathsInElf) right after the ld-linux file is/would be copied:
Not writing rpath in /home/juus/RCWork/AppImageDEV/libappimage_hooks_CDoo/commandoo.AppDir/usr/lib64/ld-linux-x86-64.so.2 because its name starts with ld-...

And the last reference to the ld-linux file is this line:
Relative path from ld-linux to Qt prefix directory in the AppDir: ../usr/lib/x86_64-linux-gnu/qt5
Which is correct for appdir/lib64, but not appdir/usr/lib64

And think about this: At this point in the code the ld-linux file has already been copied and patched. Why would another (unpatched) copy somewhere else be of use?

I am not as knowledgeable about the linux file jungle as you are. Does it have knock on effects? I don't know, I've only seen this on my system, and in this case the "superfluous" copy is a direct copy of the same ld-linux in appdir/lib64, just not patched. I'm assuming this /usr/... copy got into deployELF because it was referenced by some other library being processed as a dependency. But we all know how assumptions work out...

So, bottom line: I guess I can not promise that it will not break something. Maybe in that case just leave it alone?

**Comment by probonopd** [comment]
Well, the truth is I am not working with Linux much anymore these days. So I never came around to testing this systematically. If someone has some time, systematic testing of this topic would be highly appreciated. Thanks!
